deploy/config/crds/submarineraddon.open-cluster-management.io_submarinerconfigs.yaml

```diff
@@ -111,40 +189,66 @@ spec:
+                    description: Source represents the catalog source of a submariner
+                      subscription. The default value is redhat-operators
                     type: string
                   sourceNamespace:
                     default: openshift-marketplace
-                    description: SourceNamespace represents the catalog source namespace of a submariner subscription. The default value is openshift-marketplace
+                    description: SourceNamespace represents the catalog source namespace
+                      of a submariner subscription. The default value is openshift-marketplace
                     type: string
                   startingCSV:
-                    description: StartingCSV represents the startingCSV of a submariner subscription.
+                    description: StartingCSV represents the startingCSV of a submariner
+                      subscription.
                     type: string
                 type: object
             type: object
           status:
             description: Status represents the current status of submariner configuration
             properties:
               conditions:
-                description: Conditions contain the different condition statuses for this configuration.
+                description: Conditions contain the different condition statuses for
+                  this configuration.
                 items:
-                  description: "Condition contains details for one aspect of the current state of this API Resource. --- This struct is intended for direct use as an array at the field path .status.conditions.  For example, type FooStatus struct{     // Represents the observations of a foo's current state.     // Known .status.conditions.type are: \"Available\", \"Progressing\", and \"Degraded\"     // +patchMergeKey=type     // +patchStrategy=merge     // +listType=map     // +listMapKey=type     Conditions []metav1.Condition `json:\"conditions,omitempty\" patchStrategy:\"merge\" patchMergeKey:\"type\" protobuf:\"bytes,1,rep,name=conditions\"` \n     // other fields }"
+                  description: "Condition contains details for one aspect of the current
+                    state of this API Resource. --- This struct is intended for direct
+                    use as an array at the field path .status.conditions.  For example,
+                    type FooStatus struct{     // Represents the observations of a
+                    foo's current state.     // Known .status.conditions.type are:
+                    \"Available\", \"Progressing\", and \"Degraded\"     // +patchMergeKey=type
+                    \    // +patchStrategy=merge     // +listType=map     // +listMapKey=type
+                    \    Conditions []metav1.Condition `json:\"conditions,omitempty\"
+                    patchStrategy:\"merge\" patchMergeKey:\"type\" protobuf:\"bytes,1,rep,name=conditions\"`
+                    \n     // other fields }"
```

Comment:
Are you doing these line re-wraps or is some automation doing it?

Reply:
it is auto-generated.